app/src/main/java/org/oppia/android/app/databinding/ImageViewBindingAdapters.java

```diff
@@ -43,6 +46,27 @@ public static void setImageDrawable(
     imageView.setImageResource(drawableResourceId);
   }
 
+  /**
+   * Allows binding drawables to an [ImageView] via "android:src".
+   * Reference: https://stackoverflow.com/a/35809319/3689782.
+   */
+  @BindingAdapter("android:src")
+  public static void setImageDrawable(
+      @NonNull ImageView imageView,
+      ChapterPlayState chapterPlayState
+  ) {
+    if (chapterPlayState == ChapterPlayState.COMPLETED) {
```

Comment:
Suggest converting this if/else block to a switch.

Reply:
Updated by using ChapterPlayState